### DISCUSSION TITLE
Question : Lora Directory Change ?

### DISCUSSION THREAD

**Discussion by exodia52** [vladmandic/sdnext#119] [Q&A] [answered]
Hi, Where can i change the Lora Directory ? Couldn't find it on the Setting > System Path. If manually which file contain the directory change for the Lora ?

![image](https://user-images.githubusercontent.com/121444875/231911715-24bc96cf-f2d0-424f-8454-b32503fe265b.png)

**Comment by vladmandic** [comment]
i don't think its configurable at the moment, i'll take a look at that.

**Comment by exodia52** [comment, marked as answer]
Lora Directory is available now at settings for those who searching. Closing this.
